examples/resources/iam_policy/resource_iam_cluster.tf

```diff
@@ -0,0 +1,29 @@
+# Create Role Bindings for Cluster Scope
```

Comment:
can you add two lines of explanation here about the resource. Follow the changes done by @ishangupta-ds